### PR TITLE
improvement: Remove custom SingleStoreDBRetriever

### PR DESCRIPTION
**Pull request by volodymyr-memsql**
The SingleStoreDB vector store does not need custom logic for the retriever, so the logic from the base class should be used.

### CODE REVIEW

File: libs/langchain/langchain/vectorstores/singlestoredb.py [hunk 1]
```diff
@@ -443,32 +437,3 @@ def from_texts(
...
         instance.add_texts(texts, metadatas, embedding.embed_documents(texts), **kwargs)
         return instance
-
-    def as_retriever(self, **kwargs: Any) -> SingleStoreDBRetriever:
-        tags = kwargs.pop("tags", None) or []
-        tags.extend(self._get_retriever_tags())
-        return SingleStoreDBRetriever(vectorstore=self, **kwargs, tags=tags)
-
-
-class SingleStoreDBRetriever(VectorStoreRetriever):
```
Comment (baskaryan) [inline libs/langchain/langchain/vectorstores/singlestoredb.py:453]: for backwards compatibility can we keep alias of this class?
```python
SingleStoreDBRetriever = VectorStoreRetriever
```

**Comment by volodymyr-memsql** [comment]
@baskaryan, could you please check it again? Thanks!

**Comment by volodymyr-memsql** [comment]
Or maybe, @eyurtsev and @hwchase17 could check this?

**Comment by volodymyr-memsql** [comment]
@baskaryan, could you please merge the change? Than you!

**Comment by volodymyr-memsql** [comment]
@baskaryan, @eyurtsev, @hwchase17, could you please merge this one (or point me to some changes that should be done before)? This PR is already 2 weeks old, and the custom retriever does not support some obvious use-cases (this is what I want to fix).

Thanks!

**Pull request closed without merging**